package.json

```
@@ -30,5 +30,9 @@
   "dependencies": {
     "csvtojson": "2.0.10",
     "ical-generator": "1.9.4"
+  },
+  "devDependencies": {
+    "axios": "^0.26.0",
+    "markdown-link-extractor": "^1.3.1"
```

specify the exact version numbers, e.g.

```suggestion
    "axios": "0.26.0",
    "markdown-link-extractor": "1.3.1"
```